configuration/wireless/access-point.md

```diff
@@ -71,9 +71,9 @@ So for `wlan0`, we are going to provide IP addresses between 192.168.4.2 and 192
 
 There are many more options for dnsmasq; see the [dnsmasq documentation](http://www.thekelleys.org.uk/dnsmasq/doc.html) for more details.
 
-Reload `dnsmasq` to use the updated configuration:
+Start `dnsmasq` (it was stopped), it will use the updated configuration:
```

Comment:
It is enough to say "Start dnsmasq:" - I don't think you need the rest - I would keep it simple.